components/feature/app-links/src/test/java/mozilla/components/feature/app/links/AppLinksUseCasesTest.kt

```diff
@@ -209,11 +209,11 @@ class AppLinksUseCasesTest {
     }
 
     @Test
-    fun `A blob url is not an app link`() {
```

Comment:
This test had a copy/paste error and was testing about urls. Since it's not a blocked scheme I removed it.

Reply:
Agreed, thanks,